chat-commands.js

```diff
@@ -3209,7 +3206,7 @@ exports.commands = {
 		if (!this.can('hotpatch')) return false;
 		Rooms.global.send('|refresh|');
 		const logRoom = Rooms('staff') || room;
-		logRoom.roomlog(`${user.name} used /refreshpage`);
+		logRoom.roomlog(`${user.name} used /refreshpage.`);
```

Comment:
There should not be a period here; the command is ``/refreshpage``, not ``/refreshpage.``